src/corona_hakab_model/consts.py

```diff
@@ -76,6 +76,24 @@ class Consts(NamedTuple):
     # [10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35],
     # [0.013,0.016,0.025,0.035,0.045,0.053,0.061,0.065,0.069,0.069,0.065,0.063,0.058,0.053,0.056,0.041,0.040,0.033,
     # 0.030,0.025,0.020,0.015,0.015,0.015,0.010,0.010]))
+    # state machine transfer probabilities
+    # probability of '...' equals (1 - all other transfers)
+    # it should always come last after all other transition probabilities were defined
+    latent_to_asymptomatic_prob = 0.3
```

Comment:
add those variables changes to simulation_parameters_example.py

Reply:
done